src/handlers/shared/generate-assignment-comment.ts

```diff
@@ -15,11 +15,11 @@ export async function generateAssignmentComment(context: Context, issueCreatedAt
   let endTime: null | Date = null;
   let deadline: null | string = null;
   endTime = new Date(startTime + duration * 1000);
-  deadline = endTime.toLocaleString("en-US", options);
+  deadline = endTime.toLocaleString("en-US", options)
```

Comment:
@Keyrxng I believe this is not intended?

Reply:
Oop yeah not intended, forgot to run format after deciding to adjust the deadline elsewhere

Reply:
@Keyrxng Can you please fix this? I am fine with the PR otherwise 😄